src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. The variable 'cn' could be confusing for another developer who reads your code. Consider renaming it to something more descriptive.